### ISSUE TITLE
Doubt: is peru able to list the hashes for the git module?

### ISSUE THREAD

**Issue by felipefoz** [buildinspace/peru#229]

**Comment by oconnor663** [comment]
I don't think so, but I'm also not sure whether I'm understanding your
question correctly. Could you give me an example of output that you'd like
to see?

**Comment by felipefoz** [comment]
Ahh, sorry, I pressed the enter and it went through without my description.

In fact, the idea was to use some outputs like `peru module list` and output the module name and the hash, but that would be more of a convenience than a real need.

Later on, reading the documentation I found out that I can achieve almost the same thing, using the `reup` command and option in peru.yaml, then parse the peru.yaml file.

I say almost, because `reup` also moves to the latest hash on that tag/branch, but do not show the current status of the modules, and the latter is not my use case.

So, in summary, it would be nice to list that, but not really needed, as there are workarounds.
